store/cockroach/cockroach.go

```diff
@@ -40,44 +40,44 @@ func (s *sqlStore) Init(opts ...store.Option) error {
 }
 
 // List all the known records
-func (s *sqlStore) List() ([]*store.Record, error) {
+func (s *sqlStore) List(opts ...store.ListOption) ([]string, error) {
 	rows, err := s.db.Query(fmt.Sprintf("SELECT key, value, expiry FROM %s.%s;", s.database, s.table))
```

Comment:
Please move all queries to init phase.to avoid allocating strings and db stuff

Reply:
Refactor later?

Reply:
makes sense